### PR TITLE
[android]  - changelog for 8.5.0-alpha.1

### PR DESCRIPTION
**Pull request by tarigo**

### CODE REVIEW

File: platform/android/CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,13 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 8.5.0-alpha.1 - October 3, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.4.0...android-v8.5.0-alpha.1) since [Mapbox Maps SDK for Android v8.4.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.4.0):
 ### Bug fixes
  - Suppress network requests for expired tiles update, if these tiles are invisible. [#15741](https://github.com/mapbox/mapbox-gl-native/pull/15741)
+ - Fixed opacity interpolation for composition expressions [#15738](https://github.com/mapbox/mapbox-gl-native/pull/15738)
+### Other Changes
+ - Remove projected meters per pixel and pixel ratio relation [#15742](https://github.com/mapbox/mapbox-gl-native/pull/15742)
```
Comment (tobrun) [inline platform/android/CHANGELOG.md]: this can be placed under bug fixes

Comment (tarigo) [reply platform/android/CHANGELOG.md]: Done.

File: platform/android/CHANGELOG.md [hunk 2]
```diff
@@ -4,8 +4,12 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 8.5.0-alpha.1 - October 3, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.4.0...android-v8.5.0-alpha.1) since [Mapbox Maps SDK for Android v8.4.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.4.0):
 ### Bug fixes
  - Suppress network requests for expired tiles update, if these tiles are invisible. [#15741](https://github.com/mapbox/mapbox-gl-native/pull/15741)
+ - Fixed opacity interpolation for composition expressions [#15738](https://github.com/mapbox/mapbox-gl-native/pull/15738)
+ - Remove projected meters per pixel and pixel ratio relation [#15742](https://github.com/mapbox/mapbox-gl-native/pull/15742)
```
Comment (LukasPaczos) [inline platform/android/CHANGELOG.md]: Actually, there's a suggested changelog entry in the PR https://github.com/mapbox/mapbox-gl-native/pull/15742#issuecomment-537448116

Comment (tarigo) [reply platform/android/CHANGELOG.md]: Changed.

**Comment by LukasPaczos** [comment]
https://github.com/mapbox/mapbox-gl-native/pull/15700 would be worth including as well.

**Comment by tarigo** [comment]
> #15700 would be worth including as well.

https://github.com/mapbox/mapbox-gl-native/commit/fb2413186c7994092e5e6553dd8104802d33581f#diff-303d773d7593cfbdd7331ce896ef36afR14

**Pull request merged**